.github/workflows/ci-pr.yml

```diff
@@ -135,9 +66,23 @@ jobs:
       - name: 'Built'
         shell: bash
         run: echo "::info ::Built"
+  scan:
```

Comment:
Let's remove this scan from the PR build as we do not accept dep. updates in PRs and Trivy is failing 99% of the time due to its rate limiting.